src/components/pages/TaskType.vue

```diff
@@ -805,6 +807,20 @@ export default {
       }
     },
 
+    async loadDaysOff() {
+      this.daysOffByPerson = []
+      await Promise.all(
+        this.scheduleTeam.map(async person => {
```

Comment:
This could load to a lot of requests.

Reply:
This is limited to assigning people to the same task type.
But can I run in sequence to reduce server load, instead of parallel maybe?